app/views/cars/index.html.erb

```diff
@@ -1,7 +1,8 @@
 <% provide(:title, "My cars") %>
+<% provide(:adding_button_text, "Add new car") %>
 <ul class="list-unstyled">
   <button class="button">
-    <%= link_to "Add new car", new_car_path %>
+    <%= link_to yield(:adding_button_text), new_car_path %>
```

Comment:
what about all other strings?

Reply:
Other strings would be changed in other commits as the part of further refactorization process. This PR is focused on "adding cars" functionality and related parts of the system.

Reply:
I do not agree with this but approve. 

So why not make one thing at a time just good? Extracting strings should be basic work in new feature, like naming variables etc.